### PR TITLE
[SINGULAR] PEPPER-51: Automate Singular test case for "My child NON-assenting" enrollment

### PR DESCRIPTION
**Pull request by juanseba21**
Created e2e tests for the flow described in https://broadworkbench.atlassian.net/browse/PEPPER-51

### CODE REVIEW

File: playwright-e2e/data/fake-user.json [hunk 1]
```diff
@@ -51,5 +51,51 @@
...
+  "secondChild": {
+    "firstName": "E2E-second-Child",
+    "middleName": "Junior",
+    "lastName": "Playwright",
+    "birthDate": {
+      "MM": "07",
+      "DD": "20",
+      "YYYY": "2016"
+    },
+    "age": "06",
```
Comment (aweng98) [inline playwright-e2e/data/fake-user.json]: Any age under 7, right? I think it'd be good to make child age smaller than 6 so this new test won't fail next year.

**Review by aweng98 — CHANGES_REQUESTED** [review]
I'd recommend that this new test to be in a separate test file by itself so we don't have a very long `child-enrollment.spec.ts` test. For the new test filename, how about `child-non-assenting-enrollement.spec.ts`.

File: playwright-e2e/data/fake-user.json (hunk 1 above)
Comment (juanseba21) [reply playwright-e2e/data/fake-user.json]: done

**Review by aweng98 — DISMISSED** [review]
LGTM! Thank you.

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await medicalRecordReleaseForm.next();
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 3 of 3');
+    await medicalRecordReleaseForm.parentName().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await medicalRecordReleaseForm.parentSignature().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    //await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
```
Comment (alextanlan) [inline playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts]: Do we need the commented line? (see also line 125)

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+    const statusCell = await table.findCellLocator('Title', 'Consent Form for Minor Dependent', 'Status');
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    await expect(statusCell!).toHaveText('Complete');
+
+    await expect(page.locator('.enrollmentStatusCompleteText')).toHaveText('Fully Enrolled');
+  });
+
+  test('enrolling non assenting child > 7 not capable of consent @enrollment @singular', async ({ page }) => {
+    // Assertion helper functions
+    const assertActivityHeader = async (page: Page, expectedText: string | RegExp) => {
```
Comment (alextanlan) [inline playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts]: seems like a duplication of lines 27-33. 
I suggest extracting it like [here  ](https://github.com/broadinstitute/ddp-angular/pull/1830/files#diff-30eae4db7fabeff5de10196b1e171cd10752e48c860ee8cfc03a3c6e4ce9e09e:~:text=%40%40%20%2D0%2C0%20%2B1%2C9%20%40%40-,import%20%7B%20expect%2C%20Page%20%7D%20from%20%27%40playwright/test%27%3B,%7D%3B,-Footer).

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await medicalRecordReleaseForm.next();
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 3 of 3');
+    await medicalRecordReleaseForm.parentName().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await medicalRecordReleaseForm.parentSignature().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    //await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
```
Comment (alextanlan) [inline playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts:267]: Do we need it? (see line 290 well)

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts (hunk 3 above)
Comment (juanseba21) [reply playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts]: done

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts (hunk 4 above)
Comment (juanseba21) [reply playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts:267]: done

File: playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts (hunk 2 above)
Comment (juanseba21) [reply playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts]: done

**Review by alextanlan — APPROVED** [review]

File: playwright-e2e/utils/assertion-helper.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { expect, Page } from '@playwright/test';
+
+export const assertActivityHeader = async (page: Page, expectedText: string | RegExp) => {
```
Comment (aweng98) [inline playwright-e2e/utils/assertion-helper.ts:3]: Good idea. 👍🏻👍🏻

**Review by aweng98 — APPROVED** [review]

**Pull request merged**